dags/multipod/configs/maxtext_gce_config.py

```diff
@@ -44,18 +47,27 @@ def get_maxtext_nightly_config(
       dataset_project=project_name,
       composer_project=project_name,
   )
-  current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
-  run_name = (
-      f"{num_slices}slice-{tpu_version}_{tpu_cores}-maxtext-nightly-{current_datetime}"
-  )
 
-  set_up_cmds = common.download_maxtext()
+  current_time = datetime.datetime.now()
+  current_date = current_time.strftime("%Y-%m-%d")
+  current_datetime = current_time.strftime("%Y-%m-%d-%H-%M-%S")
+
+  if automated_test:
+    base_output_directory = f"{gcs_bucket.XLML_OUTPUT_DIR}/maxtext/{test_mode.value}/automated/{current_date}"
+  else:
+    base_output_directory = (
+        f"{gcs_bucket.XLML_OUTPUT_DIR}/maxtext/{test_mode.value}/manual/{current_date}"
+    )
+
+  run_name = f"{num_slices}slice-V{tpu_version.value}_{tpu_cores}-maxtext-{test_mode.value}-{current_datetime}"
```

Comment:
cc @RissyRan, what do you think about overriding `__str__` in `TpuVersion` to return e.g. `v4-8` instead of formatting each time?

Reply:
Yeah, I think we could do that in next update. Happy to discuss more offline.